cli/fs.rs

```diff
@@ -125,23 +125,21 @@ pub fn normalize_path(path: &Path) -> String {
 }
 
 #[cfg(unix)]
-pub fn chown(path: &str, uid: u32, gid: u32) -> Result<(), ErrBox> {
+pub fn chown(path: &str, uid: u32, gid: u32) -> Result<(), OpError> {
   let nix_uid = Uid::from_raw(uid);
   let nix_gid = Gid::from_raw(gid);
   unix_chown(path, Option::Some(nix_uid), Option::Some(nix_gid))
-    .map_err(ErrBox::from)
+    .map_err(OpError::from)
 }
 
 #[cfg(not(unix))]
-pub fn chown(_path: &str, _uid: u32, _gid: u32) -> Result<(), ErrBox> {
+pub fn chown(_path: &str, _uid: u32, _gid: u32) -> Result<(), OpError> {
   // Noop
   // TODO: implement chown for Windows
-  Err(crate::deno_error::other_error(
-    "Op not implemented".to_string(),
-  ))
+  Err(OpError::other("Op not implemented".to_string()))
 }
 
-pub fn resolve_from_cwd(path: &Path) -> Result<PathBuf, ErrBox> {
+pub fn resolve_from_cwd(path: &Path) -> Result<PathBuf, OpError> {
```

Comment:
In this utility module, it seems more appropriate to use ErrBox (or even std::io::Error if possible). OpErrors are related to ops, and these are not ops. Let's avoid propagating OpError throughout the code base.